src/PaloAltoNetworks/PaloAltoNetworks.Autorest/docs/Az.PaloAltoNetworks.md

```diff
@@ -107,12 +107,24 @@ Reset counters
 ### [Save-AzPaloAltoNetworksFirewallLogProfile](Save-AzPaloAltoNetworksFirewallLogProfile.md)
 Log Profile for Firewall
 
```

Comment:
I just noticed you do not add examples and tests for four new cmdlets. These four new cmdlets are new feature of autorest.powershell v4. In v4, if there is no patch API for a resource, we will generate an update-xxx cmdlet by using get->put API to simulate a Patch API. 
I would suggest you keep those four cmdlets and add examples and tests for them. If you do not need them, please let me know and I will remove those four cmdlets through a directive.

Reply:
Hi @dolauli, we don't need below cmdlets now. If we see any adoption, we can update these in future requests. Please remove these.

'Update-AzPaloAltoNetworksFqdnListLocalRulestack', 
'Update-AzPaloAltoNetworksLocalRule',
'Update-AzPaloAltoNetworksCertificateObjectLocalRulestack'
'Update-AzPaloAltoNetworksPrefixListLocalRulestack'